common/models.tsp

```diff
@@ -37,6 +37,9 @@ model FunctionObject {
   parameters?: FunctionParameters;
 }
 
+/** Whether to enable [parallel function calling](/docs/guides/function-calling/parallel-function-calling) during tool use. */
+alias ParallelToolCalls = boolean;
```

Comment:
According to the OpenAPI spec, this should default to `true`. Is it possible to default an alias?

Reply:
Hmm... no, I don't think we can default the alias, but we can still default each instance of the alias. Let me try to update the ingestion tool for that.